### PR TITLE
Create pull request template

### PR DESCRIPTION
**Pull request by zsd4yr**
we should have a pr template to guide contributors as they submit pull requests. CoreFX does this as well.

### CODE REVIEW

**Comment by karelz** [comment]
FYI: CoreFX does not have either PullRequest nor Issue template.
I have considered it for WinForms & WPF, but wanted to wait for real need - i.e. see what people miss, then adjust for that only if needed (less template, less process = good).

I would first ask: Do we feel we need the template? (I am personally somewhere in the middle, leaning slightly towards "not yet" ... feel free to disagree)

**Comment by zsd4yr** [comment]
My mistake, I must have grabbed it from an old blob https://github.com/dotnet/corefx/community here https://github.com/dotnet/corefx/commit/42998246d70b033b0ab2dd3378656cdf84bf349f

**Comment by karelz** [comment]
@stephentoub I think you were around when we removed PR template from CoreFX. What were the main reasons? (not useful as much?)

**Comment by zsd4yr** [comment]
Perhaps then this is not useful?

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Problem
+
+Include a summary of the problem being addressed so that reviewers don't have to extract that information from an associated issue or from the commit description(s).
+
+# Solution
+
+Include a summary of the change so that reviewers don't have to extract that information from the commit description(s).
+
+Fixes #1, #2; or Related to #1, #2 if you do not which to close the issue automaticcaly (If no corresponding GitHub issue(s) exist(s), please create them)
```
Comment (sharwell) [inline PULL_REQUEST_TEMPLATE.md]: ❗️ This is going to cause every pull request to link itself to issues 1 and 2. Unless you are planning to make that issue a hall-of-shame for people who didn't read, we should probably change this. 😂

**Pull request closed without merging**